universal-login-wallet/test/integration/ui/Transfer.tsx

```diff
@@ -33,7 +33,7 @@ describe('UI: Transfer', () => {
     appPage.transfer().enterTransferDetails(receiverAddress, '1');
 
     const tokenBalance = await appPage.dashboard().getBalance(mockTokenContract, walletAddress);
-    expect(tokenBalance).to.eq('999967821000000000');
+    expect(tokenBalance).to.eq('999967463000000000');
```

Comment:
We should make a more general statement, so we don't have to update the test each time, e.g. 
`expect(tokenBalance).to.startMatch(/9999[0-9]{16}/);`